pyanaconda/installation_tasks.py

```diff
@@ -321,6 +322,17 @@ def append(self, item):
         item.parent = self
         self._list.append(item)
 
+    @synchronized
+    def append_dbus_tasks(self, module, dbus_tasks):
+        """Append DBus Tasks from a module to the TaskQueue.
+
+        :param module: DBusServiceIdentifier instance corresponding to an Anaconda DBus module
```

Comment:
I would rename it to `service_id`, so it cannot be confused with a proxy of a DBus module.

Reply:
Sounds good, changed. :)